lib/membrane_http_adaptive_stream/hls.ex

```diff
@@ -79,48 +75,47 @@ defmodule Membrane.HTTPAdaptiveStream.HLS do
   end
 
   defp build_media_playlist_path(%Manifest.Track{} = track) do
-    "#{track.header_name |> String.split(".") |> List.first()}.m3u8"
+    [track.content_type, "_", track.id_string, ".m3u8"] |> Enum.join("")
   end
 
   defp build_media_playlist_tag(%Manifest.Track{} = track) do
     case track do
       %Manifest.Track{content_type: :audio} ->
-        """
-        #EXT-X-MEDIA:TYPE=AUDIO,NAME=#{@default_audio_track_name},GROUP-ID=#{@default_audio_track_id},AUTOSELECT=YES,DEFAULT=YES,URI="audio.m3u8"
-        """
+        "#EXT-X-MEDIA:TYPE=AUDIO,NAME=\"#{@default_audio_track_name}\",GROUP-ID=\"#{@default_audio_track_id}\",AUTOSELECT=YES,DEFAULT=YES,URI=\"audio.m3u8\""
```

Comment:
I think the previous syntax was more readable

Reply:
Again I do not want '\n' character at the end of this string to be able to  list containing this string to Enum.join("\n") and make another (bigger) part of the code more readable.

Reply:
You could have just used `String.trim/1`